### PR TITLE
Extracted constants in Google Cloud Spanner receiver config validation.

### PR DESCRIPTION
**Pull request by ydrozhdzhal**

### CODE REVIEW

**Review by jpkrohling — DISMISSED** [review]

**Review by asukhyy — APPROVED** [review]

File: receiver/googlecloudspannerreceiver/config.go [hunk 1]
```diff
@@ -42,19 +47,19 @@ type Instance struct {
 }
 
 func (config *Config) Validate() error {
-	if config.CollectionInterval.Seconds() < 60 {
-		return fmt.Errorf("%v %q must be not lower than 60s, current value is %vs", config.ID(),
-			"collection_interval", config.CollectionInterval.Seconds())
+	if config.CollectionInterval.Seconds() < minCollectionIntervalSeconds {
+		return fmt.Errorf("%v %q must be not lower than %v seconds, current value is %v sseconds", config.ID(),
```
Comment (jpkrohling) [inline receiver/googlecloudspannerreceiver/config.go:51]:
```suggestion
		return fmt.Errorf("%v %q must be not lower than %v seconds, current value is %v seconds", config.ID(),
```

Comment (ydrozhdzhal) [reply receiver/googlecloudspannerreceiver/config.go]: Done

**Review by jpkrohling — APPROVED** [review]

**Comment by ydrozhdzhal** [comment]
Looks like EasyCLA job stuck?

**Review by bogdandrutu — APPROVED** [review]

**Comment by bogdandrutu** [comment]
@ydrozhdzhal if you squash the commits this may work

**Comment by ydrozhdzhal** [comment]
> @ydrozhdzhal if you squash the commits this may work

@bogdandrutu , let me do it now

**Comment by ydrozhdzhal** [comment]
@bogdandrutu , done. Check passed.

**Comment by ydrozhdzhal** [comment]
@bogdandrutu , all checks passed except 1unrelated flaky test.

**Review by kostia-hospodarysko — APPROVED** [review]

**Pull request merged**